k8s/api-svc.yaml.template.sh

```diff
@@ -0,0 +1,22 @@
+export APP=mdn-demo-$(git rev-parse --abbrev-ref HEAD)
+eval "cat <<EOF
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: ${APP}
+  name: api
+  namespace: ${APP}
+spec:
+  ports:
+  - name: http
+    port: 80
+    protocol: TCP
+    targetPort: 8000
+  selector:
+    app: ${APP}
+    heritage: deis
```

Comment:
Good catch, it's not necessary, just something I forgot to strip out when copying from another service.